nautobot/extras/plugins/views.py

```diff
@@ -48,7 +48,7 @@ def _get_plugin_data(plugin_app_config):
             "author": plugin_app_config.author,
             "author_email": plugin_app_config.author_email,
             "description": plugin_app_config.description,
-            "verison": plugin_app_config.version,
+            "version": plugin_app_config.version,
```

Comment:
*Technically* this is a breaking API change. Might be better to add "version" and keep "verison" (sic) as well until we make breaking changes for 2.0.